### PR TITLE
ffda-usb-wan-hotplug: silence SC2034 for init.d

### PR DESCRIPTION
**Pull request by grische**

### CODE REVIEW

File: ffda-usb-wan-hotplug/files/etc/init.d/gluon-usbmuxd [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/bin/sh /etc/rc.common
+# shellcheck disable=SC2034 # some variables are from /etc/rc.common
```
Comment (blocktrron) [inline ffda-usb-wan-hotplug/files/etc/init.d/gluon-usbmuxd:2]: Comment for the reasoning in the line preceding the shellcheck parameter?

Comment (grische) [reply ffda-usb-wan-hotplug/files/etc/init.d/gluon-usbmuxd:2]: Not sure I understand:
You want me to move the comment into a separate line?

Comment (blocktrron) [reply ffda-usb-wan-hotplug/files/etc/init.d/gluon-usbmuxd:2]: Correct. Move the comment one line aboce the shellcheck parameter.

Comment (grische) [reply ffda-usb-wan-hotplug/files/etc/init.d/gluon-usbmuxd]: fixed

**Pull request merged**